setup.py

```diff
@@ -0,0 +1,11 @@
+from distutils.core import setup
+
+setup(
+    name='USPSService',
+    version='0.1',
```

Comment:
Typically, you will see 3 version numbers {major}.{minor}.{patch}. I'd recommend switching to 0.1.0.